src/solver/utils/ortools_utils.cpp

```diff
@@ -31,10 +32,80 @@ namespace Antares
 {
 namespace Optimization
 {
+
+OrtoolsLogHandler::OrtoolsLogHandler(const std::string& solver_name,
+                                     const std::filesystem::path& log_directory) :
+ solver_name_(solver_name), log_directory_(log_directory)
+{
+    init();
+}
+
+OrtoolsLogHandler& OrtoolsLogHandler::operator=(const OrtoolsLogHandler& other)
+{
+    if (this == &other)
+    {
+        return *this;
+    }
+
```

Comment:
We don't want to leak solver-specific stuff here. Only one callback if possible

Reply:
it's a security issue, if the handler is copied without a special care of the FILE*

Reply:
Not sure we want `FILE*` here.

Reply:
we have to put it somewhere! COIN solvers ask for it

Reply:
https://github.com/rte-france/or-tools/blob/3c2d4af5238b2f91e15e068ab761d4b79e5850ac/ortools/linear_solver/cbc_interface.cc#L361

Reply:
https://github.com/rte-france/or-tools/blob/3c2d4af5238b2f91e15e068ab761d4b79e5850ac/ortools/linear_solver/clp_interface.cc#L422